### PR TITLE
Fix/upgrade+virtual device

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

File: src/js/helpers.test.js [hunk 1]
```diff
@@ -120,6 +121,24 @@ systemctl restart mender-client'`
...
+    expect(code).not.toMatch(/\{([^}]+)\}/);
+    code = getDemoDeviceCreationCommand(token);
+    expect(code).not.toMatch(/\{([^}]+)\}/);
+  });
+  it('should return a sane result', () => {
+    let code = getDemoDeviceCreationCommand();
+    expect(code).toMatch('./demo --client up');
+    code = getDemoDeviceCreationCommand(token);
+    expect(code).toMatch(
+      `TENANT_TOKEN='${token}'\ndocker run -it -p 85:85 -e SERVER_URL='https://localhost' \\\n-e TENANT_TOKEN=$TENANT_TOKEN mendersoftware/mender-client-qemu:latest`
```
Comment (merlin-northern) [inline src/js/helpers.test.js:137]: the port forward `85:85` is present only here, is that on purpose?

Comment (merlin-northern) [reply src/js/helpers.test.js:137]: ah ok I see it in 
```${onboardingReducerState.demoArtifactPort}:${
        onboardingReducerState.demoArtifactPort
      }
```

Comment (mzedel) [reply src/js/helpers.test.js:137]: in the tests it is static anyways, so I can expect the `85` 👌

**Review by merlin-northern — APPROVED** [review]
lgtm!

**Pull request merged**